website/inject.js

```diff
@@ -90,9 +92,8 @@ async function getMinifiedSize (pkg, name) {
   if (excludes[name]) {
     b.exclude(excludes[name])
   }
-  b.plugin('tinyify')
 
-  const bundle = await promisify(b.bundle).call(b)
+  const { code:bundle } = await promisify(b.bundle).call(b).then(buf => minify(buf.toString(), { toplevel: true }))
```

Comment:
Assigning `bundle` variable to `code`? A tad confusing, but maybe I’m just not used to `:`.

Reply:
Seeing more of these pop up in our api repo as well. I agree it took some getting used to, but it that went pretty quickly and they do serve purpose, so i'd +1 to keep them